src/bot/rules.ts

```diff
@@ -38,24 +38,34 @@ export const createAllPushProtection = async (
       'all-branch-protections-icf',
       ['~ALL'],
     )
+
+    rulesLogger.info('Branch protection created')
```

Comment:
I'd be interested to hear the logic behind when you use info level versus debug.